solara/website/pages/examples/general/custom_storage.py

```diff
@@ -0,0 +1,69 @@
+"""# Custom state storage
+
+Solara makes it easy to store state/data on the server side, scoped to a kernel, using [reactive variables](/api/reactive).
```

Comment:
```suggestion
Solara makes it easy to store states/data on the server side, scoped to a kernel, using [reactive variables](/api/reactive).
```

I'm gonna suggest changing state -> states in all of the mentions here, since data is plural. Otherwise something like "the state" or "state information" could also be used. Let me know what you prefer.